File: back/Application/DTOs/AvaliacaoEnvio.cs
namespace Application.DTOs;

public class AvaliacaoEnvioDTO
{
    public int SessaoId { get; set; }
    public int GrupoId { get; set; }
    public int AvaliadorId { get; set; }
    public AlunoDTO Avaliador { get; set; } = null!;
    public string DeviceHash { get; set; } = "";
    public IEnumerable<AvaliacaoItemDTO>? Itens { get; set; }
}

File: front/webav360/src/app/Models/AvaliacaoEnvio.ts
import { Aluno } from "./Aluno";
import { AvaliacaoItem } from "./AvaliacaoItem";

export interface AvaliacaoEnvio {
  sessaoId: number,
  grupoId: number,
  avaliadorId: number,
  avaliador: Aluno;
  deviceHash: string,
  itens: AvaliacaoItem[]
}
